internal/twirptest/snake_case_names/compatibility_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Twitch Interactive, Inc.  All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not
+// use this file except in compliance with the License. A copy of the License is
+// located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package snake_case_names
```

Comment:
I think we should just create a separate test for this instead of reusing this existing test scenario for clarity. In addition, it would allow us to test the literal names for both the service and method names. As it stands, `TestCompatibilityWithSpecCompatibleClient` only tests the literal method name and not the service name.

Reply:
I assume this comment means a green signal for adding the new method `RegisterHaberdasherServer`.

Awesome!

Reply:
What is this referring to? I don't know what `RegisterHaberdasherServer` means.

Reply:
First half of this comment. https://github.com/twitchtv/twirp/issues/244#issuecomment-666046457

While you're at it, please also go through the first half of this comment as it can completely resolve the issue.

https://github.com/twitchtv/twirp/issues/244#issuecomment-673935091

Reply:
Resolved this to move conversation to main thread.